Pythonwin/pywin/framework/app.py

```diff
@@ -442,7 +430,8 @@ def HaveGoodGUI():
 def CreateDefaultGUI(appClass=None):
     """Creates a default GUI environment"""
     if appClass is None:
-        from . import intpyapp  # Bring in the default app - could be param'd later.
+        # Bring in the default app - could be param'd later.
+        from . import intpyapp
```

Comment:
This PR doesn't look like it was auto-generated, so why this change?

Reply:
Pretty sure that's a leftover from cleaning up / reorganizing imports. I don't particularly care whether the comment is same-line or above (as long as it's not too long, and it is under 80 chars). I can revert this if you wanna reduce changes.

Reply:
I'm fine with this isolated change but in general would prefer to avoid "personal subjective" stylistic changes (whereas "automated subjective" stylistic changes by commonly used tools are a necessary evil :)

Reply:
Yeah I'd rather leave this in whatever state won't cause a change for this PR.